### PR TITLE
Small tidy ups to support Protomaps tile server

### PR DESCRIPTION
**Pull request by halocline**

### CODE REVIEW

File: demo-app/src/pages/default/index.jsx [hunk 1]
```diff
@@ -20,17 +21,16 @@ const hostedConfig = {
 // eslint-disable-next-line no-unused-vars
 const pmtilesConfig = {
   format: 'pmtiles',
-  // TODO: Add PMtiles file to CDN and update URL
-  url: 'http://localhost:8080/planet/{z}/{x}/{y}.mvt',
+  url: import.meta.env.VITE_TILE_LAYER_URL,
```
Comment (taysea) [inline demo-app/src/pages/default/index.jsx:24]: When I open the netlify preview, I can't see the maps. Do we need to add this env variable to Netlify?

**Review by taysea — COMMENTED** [review]
One small comment about the preview/env variables but otherwise looks good.

File: demo-app/src/pages/default/index.jsx (hunk 1 above)
Comment (halocline) [reply demo-app/src/pages/default/index.jsx:24]: I've added already, just after the original preview was built. I have re-deployed and verified: https://deploy-preview-91--whimsical-moxie-6e43a0.netlify.app/

Might have been cached when you first viewed, but should be coming through now.

Comment (taysea) [reply demo-app/src/pages/default/index.jsx:24]: Works!

**Review by taysea — APPROVED** [review]
Looks good!

**Pull request merged**